src/Microsoft.IdentityModel.Tokens/EventBasedLRUCache.cs

```diff
@@ -99,50 +112,12 @@ internal class EventBasedLRUCache<TKey, TValue> : IDisposable
             _map = new ConcurrentDictionary<TKey, LRUCacheItem<TKey, TValue>>(comparer ?? EqualityComparer<TKey>.Default);
             _cleanUpIntervalInMilliSeconds = 1000 * cleanUpIntervalInSeconds;
             _removeExpiredValues = removeExpiredValues;
+            _eventQueueTaskStopTime = DateTime.UtcNow;
```

Comment:
Is the reason this is set to 'UtcNow' rather than 'UtcNow + _eventQueueTaskStopTime' because the task has not been started up yet?

Reply:
Right, it is set to UtcNow at the beginning and each time when an event is added, the _eventQueueTaskStopTime will be set to DateTime.UtcNow.AddSeconds(EventQueueTaskExecutionTimeInSeconds), see the StartEventQueueTaskIfNotRunning().